src/SignalR/clients/ts/FunctionalTests/Program.cs

```diff
@@ -27,65 +29,69 @@ public static void Main(string[] args)
                 }
             }
 
-            var hostBuilder = new WebHostBuilder()
-                .ConfigureLogging(factory =>
+            var hostBuilder = Host.CreateDefaultBuilder()
```

Comment:
Why switch to the default builder pattern here rather than `new HostBuilder()`? And if you're going to use Host.CreateDefaultBuilder(), why not also use ConfigureWebHostDefautls?

Reply:
I don't know the difference, that's why :D

Reply:
`new HostBuilder()` is vanilla, un-opinionated. `Host.CreateDefaultBuilder()` is very opinionated.

If you're trying to mimic template scenarios then use CreateDefaultBuilder, otherwise use HostBuilder.

Reply:
I'd prefer sticking closer to the templates

Reply:
`ConfigureWebHostDefaults` needs a reference to `Microsoft.AspNetCore`